src/App.js

```diff
@@ -2,12 +2,26 @@ import React, { Component } from "react";
 import Header from "./components/HeaderView";
 import BoardView from "./components/BoardView";
 
+const PHOTOS_URL = "https://picsum.photos/list";
 class App extends Component {
+  state = { photos: [] };
+
+  fetchPhotoList() {
+    fetch(PHOTOS_URL)
+      .then(resp => resp.json())
+      .then(images => images.map(image => image.id))
+      .then(images => this.setState({ photos: images }));
```

Comment:
Why would the images be a state of the app? I think this should be just a pure function that is passed into the board view.

Reply:
I'm seriously stuck here. I'm not sure what the fetch function should return. I assume that an array with photo ids. I also dont't know which lifecycle methods to use in the CardView component to update the state with photo ids and then generate `cards` array. Could you help me with this?

Reply:
A function woudl fetch an array of photo objects or photo URLs. You pretty much have:
```
getPhotos(){
return []; //array of photo urls
}
``` that goes into boardview as a function so that you can push any given set of images in any way you like to the compontent without changing it. `app`s state does not need to know what photos even are.

Reply:
I moved fetching function to the BoardView.js, because only this component uses fetched data.